javascripts/discourse/initializers/dc-popup.js.es6

```diff
@@ -0,0 +1,46 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import loadScript from "discourse/lib/load-script";
+
+let popupEnabled = false;
+
+export default {
+  name: "dc-popup",
+  initialize() {
+    withPluginApi("0.8", api => {
+      api.onAppEvent("page:changed", () => {
+        if (popupEnabled) return;
+
+        loadScript(
+          "https://unpkg.com/@debtcollective/dc-popup-component@0.0.1/dist/popup-component/popup-component.js"
+        );
+
+        addWebComponent(
+          "dc-popup",
+          {
+            id: "dc-popup",
+            hero: "We've got a new look! What do you think?",
+            class: "hydrated",
+            url:
+              "https://community.debtcollective.org/t/user-experience-feedback-new-theme/4110"
```

Comment:
What about moving this to a setting? @duranmla

Reply:
We can do so.. will do it right now and update accordingly.